src/harshalRane/Tests/MiddleCharOfMaxLength.java

```diff
@@ -0,0 +1,39 @@
+package harshalRane.Tests;
+
+public class MiddleCharOfMaxLength {
+	String getMaxLength(String[] input) {
+		int maxLength = input[0].length();
+		String maxString = input[0];
+		for (int index = 1; index < input.length; index++) {
```

Comment:
good to see that you have started loop from index 1, as you have already taken max value as 0